configure.ac

```diff
@@ -361,6 +361,22 @@ case $host in
    *linux*)
      TARGET_OS=linux
      ;;
+   *dragonfly*)
```

Comment:
Where did you get those lines? I'm worried they are untested code, as I somehow suspect you don't have access to 4 operating systems to try this on ;)

I'm curious, which one is needed for MacOS?